scripts/upgrade_admin_server.sh

```diff
@@ -100,6 +100,19 @@ n.save"
 
     # Upgrade the distribution non-interactively
     zypper --no-color --releasever 12.3 ref -f
+
+    systemctl stop rabbitmq-server
+    systemctl stop couchdb
```

Comment:
Do we really need that?

Reply:
I think couchdb could start epmd, that's the reason

Reply:
oh, I didn't know that couchdb is implemented in erlang :)